docs/verification-integration.md

```diff
@@ -107,3 +107,18 @@ Here's a summary of what happens when you run Verification:
 1. Tester Pods are started. If a Tester Pod fails, the deployer Job fails.
 1. The app and its resources are removed.
 1. The test namespace is deleted.
+
+## Troubleshooting verification errors
+
+This session covers the most common errors when running `mpdev verify`.
+
+### No value for required property: \<propertyName\>
+
+This indicates that no value was provided for the property at installation time. Since the property is required, the installation cannot proceed. To solve this make sure that the property has a default value in either /data/schema.yaml or /data-test/schema.yaml. Example of setting a default value to a property:
```

Comment:
nit: reference file names in backticks (`/data/schema.yaml` and `/data-test/schema.yaml`)